hq_superset/utils.py

```diff
@@ -228,17 +226,7 @@ def _get_domain_access(domain):
         hq_permissions = response_data['permissions']
         roles = response_data['roles'] or []
 
-        # Map between HQ and CCA
-        permissions = {
-            CAN_WRITE_PERMISSION: hq_permissions["can_edit"],
-            CAN_READ_PERMISSION: hq_permissions["can_view"],
-        }
-        return permissions, roles
-
-    @staticmethod
-    def _user_has_no_access(permissions: dict):
-        user_has_access = any([permissions[p] for p in permissions])
-        return not user_has_access
+        return hq_permissions["can_view"], hq_permissions["can_edit"], roles
```

Comment:
Previous `return` on line 223 returns a dict and a list, but this returns two bools (?) and a list. It looks like you need to update line 223.

Reply:
A test that returns a status code other than 200 would be cool, because it would have flagged the problem.

Reply:
oh, great catch @kaapstorm

Reply:
@kaapstorm 

Will update!

Reply:
https://github.com/dimagi/commcare-analytics/pull/79/commits/ca7285d967dfb9790ad639442a70ad58b835a7be

Reply:
Nice! Thank you.